packages/flutter/test/material/action_icons_theme_test.dart

```diff
@@ -115,7 +115,7 @@ void main() {
 
     await tester.pumpWidget(
       MaterialApp(
-        theme: ThemeData.light(useMaterial3: true).copyWith(actionIconTheme: actionIconTheme),
+        theme: ThemeData.light().copyWith(actionIconTheme: actionIconTheme),
```

Comment:
Maybe this can be simpler?
```suggestion
        theme: ThemeData(actionIconTheme: actionIconTheme),
```